include/zcbor_common.h

```diff
@@ -84,6 +84,10 @@ do { \
 #define MIN(a, b) (((a) < (b)) ? (a) : (b))
 #endif
 
+#ifndef NAN
+#define NAN (0.0/0.0)
```

Comment:
I'm trying to figure out where this is used.

Reply:
You're right, it was for something else. Removed.